hydra-main/web/spawn2/js/modules/task.log.js

```diff
@@ -29,8 +29,10 @@ function(
             this.stdout=undefined;
         },
         url:function(){
-            var url= "http://"+this.host+":"+this.port+"/job.log?out="+(this.stdout?'1':'0')+
-            "&id="+this.jobUuid+"&lines="+this.lines+"&node="+this.node+"&runsAgo="+this.runsAgo;
+            var url= "/job/" + this.jobUuid + "/log?out="+ (this.stdout ? "1" : "0") +
+            "&lines=" + this.lines + "&node=" + this.node + "&runsAgo=" + this.runsAgo +
+            "&minion=" + this.host + "&port=" + this.port;
+
             if(!_.isUndefined(this.offset)){
                 url+="&offset="+this.offset;
```

Comment:
Is the `offset` parameter handled by the minion job.log endpoint? It's not handled by the new spawn endpoint.